server/remote/gitea/helper.go

```diff
@@ -28,24 +28,20 @@ import (
 )
 
 // helper function that converts a Gitea repository to a Woodpecker repository.
-func toRepo(from *gitea.Repository, privateMode bool) *model.Repo {
+func toRepo(from *gitea.Repository) *model.Repo {
 	name := strings.Split(from.FullName, "/")[1]
 	avatar := expandAvatar(
 		from.HTMLURL,
 		from.Owner.AvatarURL,
 	)
-	private := from.Private
```

Comment:
We should always use auth to fetch data. Github for example has quite strict rate-limits for not authenticated users. In addition it is not required to set private globally IMO this can (already is in most cases) done by the remote repo settings anyways.

Reply:
I'd like to make sure ... - let me test this first!

Reply:
On my shedule

Reply:
I can confirm: it break stuff!